.travis.yml

```diff
@@ -20,6 +22,11 @@ install:
   - npm install
 
 before_install:
+  - wget -q -O - https://dl-ssl.google.com/linux/linux_signing_key.pub | sudo apt-key add -
+  - sudo sh -c 'echo "deb http://dl.google.com/linux/chrome/deb/ stable main" >> /etc/apt/sources.list.d/google.list'
```

Comment:
This looks like it could be fragile if the vm format shifts much, but maybe that's not an issue.  A comment explaining what the above line (or maybe the whole thing) is doing would be nice.